recipes/src/main/resources/quarkus-updates/core/3.0.yaml

```diff
@@ -26,6 +26,13 @@
 #####
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: io.quarkus.updates.camel30.CamelQuarkusMigrationRecipe
+displayName: Migrate `camel3` application to `camel4.`
+description: Migrate `camel3` quarkus application to `camel4` quarkus.
+recipeList:
+  - io.quarkus.updates.camel30.CamelQuarkusMigrationRecipe
+---
+type: specs.openrewrite.org/v1beta/recipe
```

Comment:
I found this change, which is probably not needed anymore - I will verify and remove it